docs/source/Building.md

```diff
@@ -56,7 +56,7 @@ view the [Tutorial](@ref flatbuffers_guide_tutorial) and select your appropriate
 language using the radio buttons.
 
 ### Using in CMake-based projects
-If you want to use FlatBuffers in a project which already uses CMake, then a more
```

Comment:
why remove the , ? doesn't it make it less readable?

Reply:
I thought it might be incorrect usage, but I just looked it up, and actually it might be fine with the comma.  So I will put the comma back in.